frontend/src/components/ExperimentCollection/ExperimentCollection.scss

```diff
@@ -220,7 +220,14 @@
                         padding: 0 15px 15px;
                         opacity: 0.8;
                     }
+                    .counter {
+                        color: white;
+                        position: relative;
+                        top: 30vh;
+                        left: 90%;
```

Comment:
You could also use `right` here as well. Additionally, do you want the horizontal positioning to be dependent on the width of its container, or do you want it to be a constant distance to the right (or left?) edge of the container? With things like these (items positioned on the bottom right of a container), I would normally do it something like this:

```css
.container {
  position: relative;
  /* etc. */
}

.count {
  position: absolute;
  bottom: 1rem;
  right: 1rem;
}
```